src/config/backends/backend.ts

```diff
@@ -52,7 +52,7 @@ export abstract class DeviceConfigBackend {
 	): string | string[];
 
 	// Return the env var name for this config option
```

Comment:
It'd be good if you could add a comment here that says what returning `null` actually means.